feat: add electron web demo (#154)
powersync-ja/powersync-js: demos/electron-supabase-todolist/README.md

# PowerSync + Supabase Electron with rendered Web App Demo: Todo List

## Note: Beta Release

> This package is currently in a beta release.

Reviewer: ```suggestion
The [web SDK](/packages/web/) is currently in a beta release.
```
